clients/python/sysadminctl

```diff
@@ -164,12 +175,14 @@ class ModifyCommand(object):
                 pyvalue = convertToType(args.value, existing_type)
             except ValueError as err:
                 return err
-            return format_set_output(client.set(args.key, pyvalue))
+            return format_set_output(
+                    client.set(args.key, pyvalue),
+                    args.verbose)
 
 
-def format_get_output(key, response):
+def format_get_output(key, verbose, response):
```

Comment:
Same here, I would move verbose to the end of the parameter list and make in a kwarg that defaults to False. Especially since this is Python and you won't catch argument ordering problems until this actually get's run. If you change things by adding a kwarg, all existing code will still work properly.